SnapWar.xcodeproj/xcuserdata/pialeung.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -12,7 +12,7 @@
 		<key>SnapWarShareExtension.xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>1</integer>
```

Comment:
can you remove this file from version control? git rm --cached path/to/file and then add `SnapWar.xcodeproj/xcuserdata` to `.gitignore`

Reply:
done 🙆‍♀️